README.md

```diff
@@ -213,6 +213,12 @@ The name the puppet agent daemon should run as.
 
 - *Default*: puppet
 
+ssldir
```

Comment:
rebased and added path validation in https://github.com/ghoneycutt/puppet-module-puppet/issues/113
choosed 'USE_DEFAULTS' instead of undef